sympy/series/limitseq.py

```diff
@@ -45,6 +48,129 @@ def difference_delta(expr, n=None, step=1):
         raise ValueError("Step should be bounded.")
 
     if hasattr(expr, '_eval_difference_delta'):
-        return expr._eval_difference_delta(n, step)
+        result = expr._eval_difference_delta(n, step)
+        if result:
+            return result
 
     return expr.subs(n, n + step) - expr
+
+
+def dominant(expr, n):
+    """Finds the most dominating term in an expression.
+
+    if limit(a/b, n, oo) is oo then a dominates b.
+    if limit(a/b, n, oo) is 0 then b dominates a.
+    else a and b are comparable.
+
+    returns the most dominant term.
+    If no unique domiant term, then returns ``None``.
+
+    Examples
+    ========
+
+    >>> from sympy import Sum
+    >>> from sympy.series.limitseq import dominant
+    >>> from sympy.abc import n, k
+    >>> dominant(5*n**3 + 4*n**2 + n + 1, n)
+    5*n**3
+    >>> dominant(2**n + Sum(k, (k, 0, n)), n)
+    2**n
+
+    See Also
+    ========
+
+    sympy.series.limitseq.dominant
+    """
+    terms = Add.make_args(expr.expand(func=True))
+    term0 = terms[-1]
+    comp = [term0]  # comparable terms
+    for t in terms[:-1]:
+        e = (term0 / t).combsimp()
+        l = limit_seq(e, n)
+        if l is S.Zero:
+            term0 = t
+            comp = [term0]
+        elif l is None:
+            return None
+        elif l not in [S.Infinity, -S.Infinity]:
+            comp.append(t)
+    if len(comp) > 1:
+        return None
+    return term0
+
+
+def _limit_inf(expr, n):
+    try:
+        return limit(expr, n, S.Infinity)
+    except (NotImplementedError, PoleError):
+        return None
+
+
+def limit_seq(expr, n, trials=5):
+    """Finds limits of terms having sequences at infinity.
+
+    Admissible Terms
```

Comment:
This could use a `Parameters` heading to explain what `expr`, `n`, and `trials` are. The description of admissible terms could either go under the description for `expr`, or remain its own section, with the note on expr reading something like:

```
Parameters
==========
expr : Expr
    A sympy expression composed of admissible terms (see section below)
...
```
Either works, up to you.